test/wdio/lifecycle-async/cmp-a.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('lifecycle-async', function () {
+  beforeEach(() => {
+    render({
+      template: () => <lifecycle-async-a></lifecycle-async-a>,
+    });
+  });
+
+  it('fire load methods in order', async () => {
+    await $('lifecycle-async-a').waitForExist();
+
+    await browser.pause(500);
```

Comment:
Would this work?
```suggestion
    await $('lifecycle-async-a li').waitForExist();
```

Reply:
I do still need the pause here, if I just do the `await $('lifecycle-async-a li')` then the tests ends up being flaky and seems to faily around 50% of the time

Reply:
It looks like what can happen is that it stops waiting when only one `li` is on the DOM, so the check for `.length` doesn't work :/

Reply:
I raised an issue for this and will take a look eventually: https://github.com/webdriverio/expect-webdriverio/issues/1493